components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/provisioning/impl/DefaultProvisioningHandler.java

```diff
@@ -226,6 +252,7 @@ tobeDeleted claims (claims came from federated idp as null). If there is a match
                 if (log.isDebugEnabled()) {
                     log.debug("Federated user: " + username + " is provisioned by authentication framework.");
                 }
+                diagnosticLog.info("Federated user: " + username + " is provisioned by authentication framework.");
```

Comment:
**by authentication framework** is not much meaningful for diagnostic logs? 

                diagnosticLog.info("Federated user: " + username + " is provisioned localy");

Reply:
Fixed